.github/workflows/common/set-up-mdbook/action.yaml

```diff
@@ -15,10 +15,16 @@ runs:
         # Optional Plugins have to be enabled
         # use-linkcheck: true
         # linkcheck-version: "~0.7.7"
+        # add mdbook-mermaid plugin
+        use-mermaid: true
+        mermaid-version: '0.14.1'
```

Comment:
We should add the line to `/README.md` that you also need to run `cargo install mdbook-mermaid`

```
$ mdbook serve
2025-03-28 12:26:40 [INFO] (mdbook::book): Book building has started
2025-03-28 12:26:40 [WARN] (mdbook::preprocess::cmd): The command wasn't found, is the "mermaid" preprocessor installed?
2025-03-28 12:26:40 [WARN] (mdbook::preprocess::cmd): 	Command: mdbook-mermaid
```